projects/go-lib/src/lib/components/go-portal/go-portal-attach-to.directive.ts

```diff
@@ -0,0 +1,25 @@
+import {Directive, Input, OnDestroy, OnInit, TemplateRef} from '@angular/core';
```

Comment:
Should we format these on new lines for readability?
```
import {
  Directive,
  Input,
  OnDestroy,
  OnInit,
  TemplateRef
} from '@angular/core';
```

Reply:
Are our formatting standards documented anywhere?

Reply:
@eligolding not really... It's something I've been meaning to do for a while, but have not gotten around to.